src/test/resources/soop_sst_trajectory.xml

```diff
@@ -6,68 +6,849 @@
+		<attribute name="long_name" value="'quality flags for platform_course'"/>
+		<attribute name="quality_control_set" value="3."/>
+        <attribute name="quality_control_conventions" value="'BOM (SST and Air-Sea flux) quality control procedure'"/>
+        <attribute name="flag_values" value="'B, C, D, E, F, G, H, J, K, L, M, Q, S, T, U, V, X, Z'"/>
+        <attribute name="flag_meaning" value="'Value_out_of_bounds Time_not_sequential Failed_T_Tw_Td_test Failed_true_wind_recomputation_test Platform_velocity_unrealistic Value_exceeds_threshold Discontinuity Erroneous_value Suspect_value_(visual) Value_located_over_land Instrument_malfunction Pre-flagged_as_suspect Spike_in_data_(visual) Time_duplicate Suspect_value_(statistical) Step_in_data_(statistical) Spike_in_data_(statistical) Value_passed_all_tests'"/>
+       </attributes>
+    </variable>
+    <variable>
+      <name>PL_SPD</name>
+      <encoder>float</encoder>
+      <dimensions>
+        <dimension name="TIME"/>
+      </dimensions>
+      <attributes>
+       	<attribute name="standard_name" value="'platform_speed_wrt_ground'"/>
+		<attribute name="long_name" value="'speed (over ground) of measurement platform'"/>
+		<attribute name="units" value="'meter second-1'"/>
+		<attribute name="_FillValue" value="-9999.f"/>
+		<attribute name="ancillary_variables" value="'PL_SPD_quality_control'"/>
+      </attributes>
+    </variable>
+   <variable>
+      <name>PL_SPD_quality_control</name>
+      <encoder>char</encoder>
+      <dimensions>
+        <dimension name="TIME"/>
+      </dimensions>
+      <attributes>
+       	<attribute name="standard_name" value="'platform_speed_wrt_ground status_flag'"/>
+		<attribute name="long_name" value="'quality flags for platform_speed_wrt_ground'"/>
+		<attribute name="quality_control_set" value="3."/>
+        <attribute name="quality_control_conventions" value="'BOM (SST and Air-Sea flux) quality control procedure'"/>
+        <attribute name="flag_values" value="'B, C, D, E, F, G, H, J, K, L, M, Q, S, T, U, V, X, Z'"/>
+        <attribute name="flag_meaning" value="'Value_out_of_bounds Time_not_sequential Failed_T_Tw_Td_test Failed_true_wind_recomputation_test Platform_velocity_unrealistic Value_exceeds_threshold Discontinuity Erroneous_value Suspect_value_(visual) Value_located_over_land Instrument_malfunction Pre-flagged_as_suspect Spike_in_data_(visual) Time_duplicate Suspect_value_(statistical) Step_in_data_(statistical) Spike_in_data_(statistical) Value_passed_all_tests'"/>
+       </attributes>
+    </variable>
+    <variable>
+      <name>PL_WDIR</name>
+      <encoder>float</encoder>
+      <dimensions>
+        <dimension name="TIME"/>
+      </dimensions>
+      <attributes>
+       	<attribute name="standard_name" value="''"/>
+		<attribute name="long_name" value="'wind direction (relative to moving platform) in the atmosphere'"/>
+		<attribute name="units" value="'degrees (clockwise towards true north)'"/>
```

Comment:
"'degrees (clockwise towards true north)'" doesn't make sense to me. The unit of what you measure is an angle and only needs to be referenced from a starting point which is true North (by the way are you sure about that?) so 'towards' doesn't make sense to me unless I'm missing something.

The "to" or "from" information should be in the parameter's definition. For example CF uses two distinct standard names : -wind_to_direction
-wind_from_direction

to deal with measurements of the wind direction when the given direction is either towards or from.